tests/test_websitefiles.py

```diff
@@ -508,7 +508,12 @@ def test_get_version(self):
             file.get_version('3', required=True)
 
     def test_update_version_metadata(self):
```

Comment:
We can probably remove this test since the updated test no longer test what the name suggests.